spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  describe '#initialize' do
+    it 'should be N business days later' do
+      t = DcTimePresenter.parse('2016-08-29 15:00')
+      expect(DcTimePresenter.convert(t)).to be_within(1).of(t)
```

Comment:
The problem isn't freezing time but float comparison. I could either just cast both sides `to_s` or make the interval a little tighter, but I'm not sure it matters?